gimme_aws_creds/config.py

```diff
@@ -384,6 +387,19 @@ def _get_write_aws_creds(self, default_entry):
             except ValueError:
                 ui.default.warning("Write AWS Credentials must be either y or n.")
 
+    def _get_include_path(self, default_entry):
+        """ Option to include path from rolename """
+
+        ui.default.message(
+            "Do you want to include the role path from your role name ?"
```

Comment:
To be consistent with what I wrote earlier:
```
"Do you want to include full role path to the role name in AWS credential profile name?"
```

Reply:
sounds good